package-examples/function-export/README.md

```diff
@@ -0,0 +1,103 @@
+# function-export
+
+This is a package that has both a setter and a validation function.  It shows
+how you can use gatekeeper to perform checks that are run during rendering.
+Putting a validation function  into your package allows you to give package 
+consumers guidance on what acceptable configuration is while giving them 
+the freedom to customize.
```

Comment:
The name of the example pkg is misleading :). Why don't we call it `config-validation` ? and text can be reworded to say something like:

```
This example package demonstrates how you can modify the config and ensure that modifications are compliant with the policies. This package uses:

- a `mutator` function called `label-namespace` to customize (or modify) the config
- a `validator` function `gatekeeper` to ensure changes are inline with the policy 

```

Reply:
thanks for the suggestion.  Updating.  I am still going to add a little bit of contextual guidance after the steps